targets/targets.json

```diff
@@ -2740,6 +2740,58 @@
         ],
         "device_name": "STM32G071RBTx",
         "bootloader_supported": true
+    },
+	"NUCLEO_G031K8": {
+        "inherits": [
+            "MCU_STM32_BAREMETAL"
+        ],
+        "supported_form_factors": [
+            "ARDUINO",
+            "MORPHO"
```

Comment:
I thought that "MORPHO" are the connectors on bigger nucleo like 64-pin and 144-pin but this is a NANO type so it should not be listed here

Reply:
Resolved with e74a2bc3714cee7c4960c6065d6e6f2aee01e162